src/main/java/com/github/packageurl/MalformedPackageURLException.java

```diff
@@ -33,8 +33,7 @@ public class MalformedPackageURLException extends Exception {
     /**
      * Constructs a {@code MalformedPackageURLException} with no detail message.
      */
-    public MalformedPackageURLException() {
-    }
+    public MalformedPackageURLException() {}
```

Comment:
I am against this change since braces are not supposed to be on a single line, empty or not.

Reply:
This is a choice in the [palantir-java-format](https://github.com/palantir/palantir-java-format), not modifiable.

Reply:
Not modifiable? That's odd to me. I guess nothing can be done about it then. I guess there's no "Oracle" style?